src/components/avatar/Avatar.js

```diff
@@ -0,0 +1,16 @@
+import "./Avatar.css";
```

Comment:
individual css imports can be added to style.css and only style.css can be imported in app.js